docs/Apps/Metabolomic Data/Labeled LC-MSMS Workflow.md

```diff
@@ -139,6 +141,86 @@ Click "Run* to plot the fractional enrichment plot with all the labels present a
+
+![Upload Glucose label identifier file](../../img/LabeledLC-MS2Workflow/glucose_identifier_uploads_option.png) <center>**Figure 22:** Upload Glucose label identifier file</center>
+
+![Glucose Phi expressions and intermediate expressions identifier file formats](../../img/LabeledLC-MS2Workflow/glucose_phi_expressions_file_format.png)
+
+![Glucose Phi expressions and intermediate expressions identifier file formats](../../img/LabeledLC-MS2Workflow/glucose_intermediate_expression_file_format.png)<center>**Figure 23:** Glucose Phi expressions and intermediate expressions identifier file formats</center>
+
+- **Any generic label (eg: Glutamine Labeled):** In case the user has any other generic label data other than Glucose, they can make use of this option. The app will perform Phi calculation in the same way as for <sup>13</sup>C Glucose labeled elements.
+
+- **Upload your generic label identifier file:** The app by default has the identifier expressions file for glutamine label. If at all, the user has the identifier file for glutamine with additional expressions or any other generic label, they can select onto this option and further upload the identifier expression file for the same.
+
+![Upload Generic label identifier file](../../img/LabeledLC-MS2Workflow/generic_label_file_upload.png) <center>**Figure 24:** Upload Generic label identifier file</center>
+
+![Generic label Phi expressions identifier file format](../../img/LabeledLC-MS2Workflow/glutamine_identifier_file_format.png) <center>**Figure 25:** Generic label Phi expressions identifier file format</center>
+
+- **Upload your metadata mq with identifier file:** The app by default has the metadata mq identifier loaded, but in case the user has made use of any additional expressions for the above parameters, they will have to upload their own metadata mq identifier file. Keeping it unchecked would utilise the metadata mq file within the app.
+
+![Metadata mq identifier file format](../../img/LabeledLC-MS2Workflow/metadata_mq_identifier_file_format.png) <center>**Figure 26:** Metadata mq identifier file format</center>
+
+- **Subset by cohorts for glucose labelled (for TCA Cycle)**: The user can subset the specific cohorts that belong to the Glucose labelled (for TCA Cycle) to proceed with the Phi calculation. The user can type out a characteristic string that denotes the particular cohort and select onto the required cohorts from the dropdown.
+
+- **Subset by cohorts for generic labelled**: The user can subset the specific cohorts that belong to the Glucose labelled (for TCA Cycle) to proceed with the Phi calculation. The user can type out a characteristic string that denotes the particular cohort and select onto the required cohorts from the dropdown.
+
+After uploading the necessary files, click on '*Calculate Phi*'.
```

Comment:
@KMaheswari Remove the single quotation marks. If you are mentioning the exact text from the platform then it just has to be in italics.